src/dbus/server/CMakeLists.txt

```diff
@@ -36,18 +36,32 @@ add_custom_target(otbr-dbus-introspect-header ALL
 
 option(OTBR_ENABLE_LEGACY "enable legacy support")
 
+SET(PROTO_GEN_DIR ${PROJECT_BINARY_DIR}/src/proto)
+# Set the protoc output files as GENERATED to avoid the compile time check.
+set_source_files_properties(${PROTO_GEN_DIR}/feature_flag.pb.cc
+    PROPERTIES GENERATED TRUE)
+
+string(REPLACE "-Wunused-parameter" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
+
 add_library(otbr-dbus-server STATIC
     dbus_agent.cpp
     dbus_object.cpp
     dbus_thread_object.cpp
     error_helper.cpp
+    ${PROTO_GEN_DIR}/feature_flag.pb.cc
```

Comment:
Please make the `feature_flag` itself an independent CMake library rather than being part of `otbr-dbus-server`, because there will be other modules will refer to the `feature_flag`.

Reply:
Done.